Gemfile.lock

```diff
@@ -1,13 +1,18 @@
 GIT
   remote: https://github.com/wordpress-mobile/release-toolkit
-  revision: a7f414b31dc383075d3489b3d9d4ec039529ef67
-  tag: 0.1.9
+  revision: 26046e1282e9675323b71e0af12d9f070a6e57b6
+  branch: integrate-screenshot-tool
```

Comment:
This is just for the purposes of reviewing this PR. Assuming we're happy with it, I can submit a new PR to adjust this for the tag, or can tag a new release in https://github.com/wordpress-mobile/release-toolkit